libbeat/dashboards/import_dashboards.go

```diff
@@ -448,16 +448,15 @@ func (imp Importer) ImportDir(dirType string, dir string) error {
 	// check if the directory exists
 	if _, err := os.Stat(dir); err != nil {
 		// nothing to import
-		fmt.Println("No directory", dir)
-		return nil
+		return fmt.Errorf("No directory %s", dir)
```

Comment:
Ignoring the error here was also intentional because some beats (e.g. Filebeat) only have the index-pattern but not the dashboard folder. If we return an error here, we make the script always return non-zero for Filebeat.